packages/help-center/src/styles.scss

```
@@ -279,3 +279,86 @@ button.button.back-button__help-center.is-borderless {
 		border-right-color: var( --wp-admin-theme-color );
 	}
 }
+
+.is-section-checkout {
+	.help-center {
+		button,
+		input,
+		textarea,
+		select {
+			box-sizing: border-box;
+		}
+
+		h1 {
+			font-weight: 600;
+			margin: 0.67em 0;
+
+			&.site-picker__site-title {
+				font-weight: 400;
+				margin: 0;
+			}
+		}
+
+		h2,
+		h3 {
+			color: #1d2327;
```

```suggestion
			color: var( --studio-gray-90 );
```

I think this should use the variable as well. I matched it from here https://github.com/Automattic/color-studio/blob/master/dist/color-variables.scss